app.py

```diff
@@ -40,6 +40,32 @@ def mirror(name):
 
 # TODO: Implement the rest of the API here!
 
+@app.route('/users')
+def users():
+    if 'team' in request.args:
+        data = db.getByTeam('users', request.args.get('team'))
+        if data is None:
+            return create_response({})
+        return create_response(data)
+    else:
+        return create_response(db.get('users'))
+
+@app.route('/users', methods=['POST'])
+def createUser():
+    if not 'name' in request.json or not 'age' in request.json or not 'team' in request.json:
```

Comment:
For Flask, `request.get_json()` should be used in favor of `request.json`

Do `if x not in y` instead of `if not x in y` since it's a little easier to read.

You can use a [try statement](https://docs.python.org/3/tutorial/errors.html#handling-exceptions) instead of doing the long `if` statement to check if each item is in `request.json`.

If you want to check things this way with the `if` statement, then you should use the built-in `all` function:
`if all(x in request.get_json() for x in ['name', 'age', 'team'])`